shlr/lz4/xxhash.h

```diff
@@ -0,0 +1,293 @@
+/*
```

Comment:
we have xxhash already in r2, so better to r2-ify the lz4 thing if you want to put it in core

Reply:
otherwise its fine to put it in extras as a single plugin with everything inside